models/eos/ntp_global/eos_ntp_global.yml

```diff
@@ -0,0 +1,151 @@
+---
+module: eos_ntp_global
+short_description: Manages ntp resource module
+description: This module configures and manages the attributes of  ntp on Arista
+  EOS platforms.
+version_added: 3.1.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.24.60M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+   config:
+      description: A dictionary of ntp options
+      type: dict
+      suboptions:
+        authenticate:
+          description:
+          - Require authentication for NTP synchronization.
+          type: dict
+          suboptions:
+            enable:
+              description: Enable authentication for NTP synchronization.
+              type: bool
+            servers:
+              description: Authentication required only for incoming NTP server responses.
+              type: bool
+        authentication_key:
+          description:
+          - Define a key to use for authentication.
+          type: dict
+          suboptions:
+            key_identifier:
+              description: key identifier.
+              type: int
+            algorithm:
+              description: hash algorithm,
+              type: str
+              choices: ["md5", "sha1"]
+            key_type:
```

Comment:
Does this define whether the key is clear-text or encrypted? If so, can we name it `encryption`? Like https://github.com/ansible-network/resource_module_models/pull/149/files#diff-f6fc3bc94c48edafdd7ee275b4dacb2553a99690c1edb5721a395df58691274aR79-R83